next/.gitignore

```diff
@@ -26,7 +26,7 @@ yarn-debug.log*
 yarn-error.log*
 
 # local env files
-.env*.local
+.env
```

Comment:
That's my bad – recently I tell you to use `.env` instead of `.env.local`. That's not a big thing, but Next.js in their [create-next-app](https://nextjs.org/docs/app/api-reference/create-next-app) used a name `.env.local`, so that's completely fine.